adapters/e_volution/evolution.go

```diff
@@ -0,0 +1,112 @@
+package evolution
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"net/http"
+)
+
+type adapter struct {
+	URI string
+}
+
+type bidExt struct {
+	MediaType openrtb_ext.BidType `json:mediaType,omitempty`
```

Comment:
The json tag should have double quotes around the value, ie: ``json:"mediaType,omitempty"``